cmd/opera/launcher/fake.go

```diff
@@ -19,19 +19,19 @@ var FakeNetFlag = cli.StringFlag{
 }
 
 func getFakeValidatorKey(ctx *cli.Context) *ecdsa.PrivateKey {
-	num, _, err := parseFakeGen(ctx.GlobalString(FakeNetFlag.Name))
+	id, _, err := parseFakeGen(ctx.GlobalString(FakeNetFlag.Name))
 	if err != nil {
```

Comment:
I suggest to combine both cases into one:  ` if err != nil ||  id == 0 {  return nil }`